src/Core/src/Platform/Windows/ViewExtensions.cs

```diff
@@ -44,6 +44,11 @@ public static void UpdateBackground(this FrameworkElement nativeView, IView view
 
 		public static void UpdateAutomationId(this FrameworkElement nativeView, IView view) =>
 			AutomationProperties.SetAutomationId(nativeView, view.AutomationId);
+		
+		public static void UpdateOpacity(this FrameworkElement nativeView, IView view)
+		{
+			nativeView.Opacity = view.Opacity;
```

Comment:
Setting view visibility also makes use of this parameter, so the Visibility and Opacity properties need to use the same mapping method.